gnosis/eth/oracles/utils.py

```diff
@@ -32,6 +32,6 @@ def get_decimals(
         BadFunctionCallOutput,
         DecodingError,
     ) as e:
-        error_message = f"Cannot get decimals for token={token_address}"
-        logger.warning(error_message)
-        raise CannotGetPriceFromOracle(error_message) from e
+        message = f"Cannot get decimals for token={token_address}"
```

Comment:
Is this related with the price? I mean, ERC20 should have decimals method.

Reply:
Yes, but when we are calculating the price we use anything that looks like an ERC20, so I wouldn't log this kind of errors. This method is only used from `oracles`